Sources/Tracking/DeviceInfo.swift

```diff
@@ -1,15 +1,56 @@
 import Foundation
 #if canImport(UIKit)
 import UIKit
+import UserNotifications
```

Comment:
Linux doesn't support `UserNotifications` so you have to [do this](https://github.com/customerio/customerio-ios/blob/0ca06183f9c0094019e3e58d0963b5931909c348/Sources/MessagingPush/PushContent.swift#L3)

Reply:
How did I miss this one 🤔  Thanks !